app/models/user.rb

```diff
@@ -66,7 +68,11 @@ def multiple_identities?
   end
 
   def active_profile
-    profiles.where(active: true).first
+    profiles.find(&:active?)
+  end
+
+  def verified?
+    active_profile ? true : false
   end
```

Comment:
Perhaps I'm misunderstanding, but I was under the impression that it was recommended to use a separate class for business logic, as well as methods that reach beyond the model itself (such as here, where it checks for an attribute in the Profile class). Let me see if I can find some references.